less/admin/PermissionsPage.less

```diff
@@ -49,19 +50,36 @@
     color: @muted-color;
   }
   thead th {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    padding-bottom: 10px;
     text-transform: uppercase;
     font-weight: bold;
     font-size: 12px;
     color: @muted-color;
+    background: @body-bg;
     min-width: 140px;
+    z-index: 1;
+
+    &:first-child {
+      left: 0;
+      z-index: 3;
+      background: @body-bg;
+    }
```

Comment:
might be better without this as well 🤔

Reply:
this is not strictly necessary but IMO there's a real aesthetic interest: without it, the `thead` row overlaps the permission labels column when scrolling horizontally while being on top

Reply:
There may have been some overlap before, but with the updated code, removing this works nicely, I think what I like about it is that it's no longer a dead area where neither the tags nor the permission labels take when scrolling.

See here what I mean:

https://user-images.githubusercontent.com/20267363/123166758-aafa8280-d46d-11eb-9042-932a8ccbd3e7.mp4

Reply:
OK I see the point! When scrolling down I totally agree, but when you're at the top and scrolling horizontally, aren't you annoyed by this?

| with `&:first-child`... | without `&:first-child`... |
| ------------------------- | --------------------------- |
| ![Firefox_Screenshot_2021-06-24T06-56-17 919Z](https://user-images.githubusercontent.com/34077099/123217391-cf387c80-d4ca-11eb-8aa8-4170f0f7d32c.png) | ![Firefox_Screenshot_2021-06-24T06-57-03 194Z](https://user-images.githubusercontent.com/34077099/123217419-d790b780-d4ca-11eb-99b3-c88d1c681cc6.png) |

It's a detail, maybe I'm too pernickety :shipit:

Reply:
Yea, I think it's just a matter of preference really, I like being able to see the tags hidden when on top, but that might not be the case for others.

Let's just leave it open for a third opinion and we can make changes based on that!

Reply:
Roger that :+1:

Reply:
Argh, it's a hard one! I like seeing the tag because it tells me there's more options to the left, but I dislike it because it doesn't look nice.

I'd err on the side of hiding it though, if I had to choose.

Reply:
> I like seeing the tag because it tells me there's more options to the left

Maybe that's the key for a temporary UX-based decision, even if I too dislike the design :see_no_evil: I'll try to play around the previous solution asap, and see if I can figure out a compromise

Reply:
It's fine tbh, I'm not sure there can be a compromise, it's a small detail anyway, we can just bring it back 👍🏼

Reply:
Deal :slightly_smiling_face: 
I'll revert my last commit this evening (french time zone :clock10:) and I'll maybe play around a compromise for a future PR if I find a good idea.

Reply:
done :)